### ISSUE TITLE
Nuget package does not copy native libraries to build output on Windows 

### ISSUE THREAD

**Issue by henrivain** [henrivain/TesseractOcrMaui#8]
Output bin folder does not have lib -folder inside
<img width="1613" alt="image" src="https://user-images.githubusercontent.com/89461562/231261252-ced38414-a769-4901-a605-cbf08a408247.png">

Also can't find them by searching
<img width="590" alt="image" src="https://user-images.githubusercontent.com/89461562/231261664-6b1540de-cdd6-40fa-8823-05c12b40222b.png">

With project reference everything works fine, but with nuget package dll cannot be loaded.

Android works fine.

Affected platforms: Windows

**Comment by henrivain** [comment]
Added package path whilst running pack build in release mode

``` xml
<ItemGroup Condition="'$(Configuration)' == 'Release' ">
    <Resource Include="lib\Windows\x86_64\*.dll" Pack="true" PackagePath="lib\net7.0-windows10.0.19041\">
	<CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
	<PackageCopyToOutput>true</PackageCopyToOutput>
    </Resource>
</ItemGroup>
```

Also copy windows dlls to output directory for debugging repository project 
- 
``` xml
<ItemGroup Condition="$(TargetFramework.Contains('-windows'))">
    <ContentWithTargetPath Include="lib\Windows\x86_64\**">
	<CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
	<TargetPath>%(Filename)%(Extension)</TargetPath>
    </ContentWithTargetPath>
</ItemGroup>
```

Change dll import dll name to point to output dir on windows
- old was lib\Windows\x86_64\tesseract53.dll
``` cs
const string DllName = "tesseract53.dll";
```

**Comment by henrivain** [comment]
Fixed in v1.0.6
